openfasoc/generators/PMU-gen/tools/parse_rpt.py

```diff
@@ -22,3 +22,26 @@
     raise ValueError("LVS failed!")
 else:
     print("LVS is clean!")
+
+# with open('test.json', 'r') as file:
```

Comment:
can we remove this comments part please? I don't know when this generator work will be resumed and this might confuse people who will be developing this in future. maybe we can keep this file like it was ?